### DISCUSSION TITLE
How to set the number.solaredge_active_power_limit using actions

### DISCUSSION THREAD

**Discussion by Pat28** [binsentsu/home-assistant-solaredge-modbus#164] [Q&A] [answered]
I am using this

`service: input_number.set_value
target:
  entity_id: number.solaredgemodbus_active_power_limit
data:
  value: 10
`

But nothing is changed.
While dragging the slider does work and the inverter changes (can be seen on the LCD screen),
so the integration works, but how to get this automated.

**Comment by binsentsu** [comment, marked as answer]
service should be number.set_value iso input_number.set_value
